implementations/micrometer-registry-prometheus_native/gradle.properties

```diff
@@ -0,0 +1 @@
+compatibleVersion=SKIP
```

Comment:
where is the version number?

Reply:
This is for the API/ABI compatibility check. Since there isn't a previous version of this module to compare with, this configuration skips the compatibility check so the build doesn't fail.